docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm

```diff
@@ -53,6 +53,7 @@ <h2>Table of Contents</h2>
 <h2 id="manual-constructor">How to Create Section Changer</h2>
 
 <p>To add a section changer component to the application, use the following code:</p>
+<p>HTML code:</p>
 <pre class="prettyprint"><pre class="prettyprint lang-html" style="border:0px; margin:0px">
```

Comment:
Is this `<pre class="prettyprint"><pre class="prettyprint lang-html" style="border:0px; margin:0px">` intended? 2 `<pre>` tags are duplicated. Related with `&lt;/div&gt;</pre>` on line 76 and `</pre>` on line 78.

Reply:
Fixed